cloudsql-mysql-plugin/src/e2e-test/features/source/CloudMySqlDesignTime.feature

```diff
@@ -0,0 +1,74 @@
+# Copyright © 2023 Cask Data, Inc.
+##
+## Licensed under the Apache License, Version 2.0 (the "License"); you may not
+## use this file except in compliance with the License. You may obtain a copy of
+## the License at
+##
+## http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+## WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+## License for the specific language governing permissions and limitations under
+# the License..
+
+@CloudMySql
```

Comment:
Should this also be a required test (@CloudMySql_Required) similar to the one for sink (cloudsql-mysql-plugin/src/e2e-test/features/sink/CloudMySqlDesignTime.feature)?

Reply:
The Required tag has been included for every feature of the source and sink. Added the @CloudMySql_Required tag to this feature.